src/main/java/com/stripe/model/Charge.java

```diff
@@ -463,7 +463,7 @@ public static Charge retrieve(String id) throws StripeException {
    * Retrieve a charge.
    */
   public static Charge retrieve(String id, RequestOptions options) throws StripeException {
-    return request(RequestMethod.GET, instanceUrl(Charge.class, id), null, Charge.class, options);
+    return retrieve(id, null, options);
```

Comment:
Minor but you end up piping through 3 methods instead of 2 because the one with just the id will call that one which then calls the one with 3 parameters. Definitely not a big deal but wanted to mention it